### ISSUE TITLE
`R_user_dir` only points to global library?

### ISSUE THREAD

**Issue by matt-dray** [matt-dray/tamRgo#33]
I was hoping that a new {renv}-backed project would allow a 'contained' instance of a new pet, but I get the same path from `R_user_dir()` whether I run it in (a) a project with access to my computer's global R-package library (i.e. not {renv}-backed), or (b) within a contained, {renv}-backed project that has {tamRgo} freshly installed. 

This seems to be true even if my global install is an earlier version than in a {renv}-backed project (e.g. I tried 9004 and 9005, respectively).

In other words, both projects point to:

```
tools::R_user_dir("tamRgo", "data")
[1] "/Users/matt.dray/Library/Application Support/org.R-project.R/R/tamRgo"
```

Possibly I'm misunderstanding how one or all of these actually work.
